interact.css

```diff
@@ -0,0 +1,78 @@
+@media only screen and (max-width: 768px) {
+  .header-mobile {
+    color: #0069d9;
+  }
+
+  .open .toolbar {
+    background-color: hsl(234, 100%, 69%);
+    padding-top: 20px;
+    position: fixed;
+    top: 0;
+    opacity: 0.93;
+    z-index: 9999;
+    height: 100vh;
+    width: 100%;
+  }
```

Comment:
- [ ] Kindly give enough spacing between the links in the mobile menu. This would make your design identical to that of the Figma design chosen. 👍 😎 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/225133548-ad356e0b-3e90-4493-8c73-bec7d923c630.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/225133636-08599413-d2b2-44bc-b038-58b2e81c4721.png)